src/utilities/exec.ts

```diff
@@ -0,0 +1,33 @@
+import * as aexec from '@actions/exec';
+
+export interface ExecResult {
+  success: boolean;
+  stdout: string;
+  stderr: string;
+}
+
+export const exec = async (command: string, args: string[] = [], silent?: boolean): Promise<ExecResult> => {

```

Comment:
Consider using `ToolRunner` instead of `exec`

Reply:
Made the required changes for this